main.py

```diff
@@ -39,28 +53,53 @@ class ShoppingCart:
     ``ShoppingCart`` instance should implement a method to calculate
    the total price of entire cart.
     """
-    products_in_cart = []
+    def __init__(self):
+        products_in_cart = []
```

Comment:
Нащо ці рухи туди-сюди?
```suggestion
        self.products = []
```